nixos/doc/manual/release-notes/rl-2505.section.md

```diff
@@ -99,6 +99,8 @@
 
 - [git-worktree-switcher](https://github.com/mateusauler/git-worktree-switcher), switch between git worktrees with speed. Available as [programs.git-worktree-switcher](#opt-programs.git-worktree-switcher.enable)
 
+- [rustic](https://github.com/rustic-rs/rustic), a Rust-based backup tool. Available as [services.rustic](#opt-services.rustic.enable).
```

Comment:
As this module is quite complex, IMO it necessitates some unit tests.

Reply:
Makes sense, I actually just got some motivation/time and added basic unit tests, validating at least the main functionality (including prometheus metrics generation, though checking the exact syntax would be hard) :)